edx_proctoring/api.py

```diff
@@ -63,7 +65,8 @@
 
 
 def create_exam(course_id, content_id, exam_name, time_limit_mins, due_date=None,
-                is_proctored=True, is_practice_exam=False, external_id=None, is_active=True, hide_after_due=False):
+                is_proctored=True, is_practice_exam=False, external_id=None, is_active=True, hide_after_due=False,
+                backend=getattr(settings, 'PROCTORING_BACKENDS', {}).get('DEFAULT', None)):
```

Comment:
You've isolated yourself from settings that might not be available, but don't we want to read the settings at the time the function is called? Why are we working so hard to squeeze this into a parameter default?

Reply:
I moved it into the function.